content/components/selectpanel.mdx

```diff
@@ -9,51 +9,155 @@ import {Box} from '@primer/react'
+
+For errors we should always include a link to our support page to ensure that users have a way to reach out to us for assistance.
+
+### Empty state
+
+<img
+  width="960"
+  alt="Error"
+  src="https://github.com/primer/design/assets/980622/57386be3-6e16-4862-b557-d14533b3ef28"
+/>
+
+When no results are available, it is essential to offer users an explanation instead of leaving a blank space. There can be various reasons for the absence of results. It could be due to the search or filter input value not matching any available data, or it could be because the user has yet to generate any data to display.
+
+### Limit selections
+
+<img
+  width="960"
+  alt="First select panel is a branch switcher control that includes tabs, a search input, a listbox with single-select items, and a footer with an All branches link, and cancel and apply buttons. Second select panel is titled 'Show fields, with a list of about 10 multi-select items and apply and cancel buttons in the footer."
+  src="https://github.com/primer/design/assets/980622/9cacdd30-99b3-4571-b37f-3066b7219ab2"
+/>
+
+In certain situations, we may need to restrict the number of items a user can select, either due to infrastructure limitations or limitations based on paid features. It is important to inform the user about this restriction before they begin making their selection. The description can be used effectively to communicate this limitation.
+
+When the user reaches the maximum allowed number of selections, we need to display a warning message indicating that they have reached their limit. This can also be an opportunity to provide a link for upgrading their account if the restriction is related to a paid feature. It is worth noting that the warning message is labeled with an `aria-live` attribute, ensuring that users with screen readers receive a notification when the message appears.
+
+### Loading
+
+It is important to provide visual cues to the user when certain processes may take longer than expected. Loading states can be utilized at different stages of using this component to communicate this information effectively.
+
+<img
+  width="1668"
+  alt="First select panel is a branch switcher control that includes tabs, a search input, a listbox with single-select items, and a footer with an All branches link, and cancel and apply buttons. Second select panel is titled 'Show fields, with a list of about 10 multi-select items and apply and cancel buttons in the footer."
+  src="https://github.com/primer/design/assets/980622/52bcce0e-bed4-44e8-83b4-8ca6bf80eb57"
```

Comment:
We might not want to include avatars in the skeleton. I'm not sure we'll be able to know if the items have avatars or not.

Reply:
@ichelsea what are the accessibility concerns with skeleton loaders? Could we perhaps always use skeleton loaders? This would simplify the API and creates more consistency?